PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- Open this PR as draft while it is not ready -->
```

Comment:
In `Quickstart Android Jetpack Graphql`, we have a comment in the beginning that I think it's nice to have, or add at PR checklist section. What do you guys think?

```

<!--
BEFORE opening a PR, double check the following items:

IMPORTANT
- [ ] I've updated/created the necessary documentation related to my PR
- [ ] I've created test cases to cover my changes (if applicable)

GITHUB
- [ ] I've changed the title of my PR
- [ ] I've assigned the PR to myself

If all of the items above are checked, feel free to open your PR 
-->

```

Reply:
Sounds good to me

Reply:
@mari1912 This comments serves the same purpose as the `#### Pull request checklist` section. I prefer this checklist not to be a comment because the contributor may create a WIP PR without, or with incomplete tests and documentation. And after the PR is opened, s/he will no longer see this checklist and may forget it. 

Mozilla's contribution template also uses a non-commented PR checklists.

Reply:
Looks good